### DISCUSSION TITLE
For adjust the information of IAP

### DISCUSSION THREAD

**Discussion by 954708775** [fastlane/fastlane#20447] [Q&A]
With the help of Fastlane, I could configure the IAPs automatically via Ruby. But I am wondering is it possible to adjust the information of the passed review IAPS, such as the addition of new localization, or the price tier change. Hope someone may help me with the issue.

**Comment by WackyShady** [comment]
It seems that fastlane cannot work now, my permisson for apple store connect is lost via Ruby
